plugins/c8y_log_plugin/src/main.rs

```diff
@@ -95,14 +99,16 @@ async fn run(
     mqtt_client: &mut Connection,
     http_client: &mut JwtAuthHttpProxy,
 ) -> Result<(), anyhow::Error> {
-    let mut plugin_config = handle_dynamic_log_type_update(mqtt_client, config_file).await?;
-
+    let mut plugin_config = LogPluginConfig::default();
```

Comment:
The fact that a call to `handle_dynamic_log_type_update` can be replaced by a call to `handle_dynamic_log_type_update` raises a warning. The `handle_dynamic_log_type_update` is doing to many things. Reading the config and notifying the cloud with a list of log types.